resources/views/device/show.blade.php

```diff
@@ -0,0 +1,136 @@
+@extends('layouts.dashboard')
+
+@section('title', 'Show Device')
+
+@section('content')
+<div class="container">
+    <div class="row">
+        <div class="col-xs-12 col-sm-12 col-md-10 col-lg-10 col-xs-offset-0 col-sm-offset-0 col-md-offset-1 col-lg-offset-1">
+            <div class="panel panel-primary">
+                <div class="panel-heading">
+                    <h3 class="panel-title">Device information</h3>
+                </div>
+                <div class="panel-body">
+                    <div class="row">
+                        <div class="col-md-3 col-lg-3 hidden-xs hidden-sm">
+                            <img class="img-thumbnail img-responsive"
+                                 src="/image/device/{{ $device->id }}"
+                                 alt="Device picture">
+                        </div>
+                        <div class="col-xs-2 col-sm-2 hidden-md hidden-lg">
+                            <img class="img-thumbnail img-responsive"
+                                 src="/image/device/{{ $device->id }}"
+                                 alt="Device picture">
+                        </div>
+                        <div class="col-md-9 col-lg-9 hidden-xs hidden-sm">
+                            <strong>{{ $device->name }}</strong><br>
+                            <table class="table table-device-information">
+                                <tbody>
+                                <tr>
+                                    <td>Location:</td>
+                                    <td>{{ $device->location_id }}</td>
```

Comment:
Can you display the location name instead of the location id.

Reply:
fixed in commit 28bda35
note the new user model method

Reply:
fixed in commit 963fc3f